workitem/undoable_wi_repo.go

```diff
@@ -125,3 +125,7 @@ func (r *UndoableWorkItemRepository) List(ctx context.Context, criteria criteria
 func (r *UndoableWorkItemRepository) GetCountsPerIteration(ctx context.Context, spaceId uuid.UUID) (map[string]WICountsPerIteration, error) {
 	return map[string]WICountsPerIteration{}, nil
 }
+
+func (r *UndoableWorkItemRepository) GetCountsForIteration(ctx context.Context, iterationId uuid.UUID) (map[string]WICountsPerIteration, error) {
```

Comment:
Do we really need these functions ? We don't even use the input arguments. The just return a map... it makes no much sense to me. The same applies to the function `GetCountsPerIteration`.